sql/catalyst/src/main/java/org/apache/spark/sql/connector/write/LogicalWriteInfo.java

```diff
@@ -45,4 +47,20 @@ public interface LogicalWriteInfo {
    * the schema of the input data from Spark to data source.
    */
   StructType schema();
```

Comment:
In delta-based DELETE operations, I plan to return an empty struct type here. See [here](https://github.com/apache/spark/pull/38004#discussion_r982723941) for context.